tools/internal_ci/macos/grpc_objc_bazel_test.sh

```diff
@@ -119,3 +119,33 @@ objc_bazel_tests/bazel_wrapper \
   -- \
   "${EXAMPLE_TARGETS[@]}" \
   "${TEST_TARGETS[@]}"
+
+
+# Enable event engine and run tests again.
+TEST_TARGETS=(
+  //src/objective-c/tests:InteropTestsLocalCleartext
+  //src/objective-c/tests:InteropTestsLocalSSL
+  //src/objective-c/tests:InteropTestsRemote
+  //src/objective-c/tests:MacTests
+  //src/objective-c/tests:UnitTests
+  //src/objective-c/tests:tvtests_build_test
+  # codegen plugin tests
+  //src/objective-c/tests:objc_codegen_plugin_test
```

Comment:
sounds like the plugin_test and plugin_option test  won't be any different under event engine (they only run the protoc code AFAICT), so you might as well skip them here?